notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>It would be helpful to add some comments to explain the different initialisation, assimilation and evaluation steps in this cell. Please also define the acronyms and symbols used (EPE, AE, grad, div, lap).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p> </p><p>This cell takes long time. It would be helpful to include timing information for each for loop. This way, the person running the code can estimate how long it will take to complete this cell. </p><p>For example at beginning of the for loop, note the time like</p><pre class="ql-syntax" spellcheck="false">time_start = time.perf_counter()&nbsp;
</pre><p>and just before the end of for loop we can print </p><pre class="ql-syntax" spellcheck="false">time.perf_counter() - time_start
</pre><p> </p>

Reply:
<p>@<a href="https://github.com/polpel" target="_blank">polpel</a> I've added some definition of the acronyms and symbols. I'll open an issue to indicate further comments or context of the steps would be beneficial to improve the readiness of the cell.</p>

Reply:
<p>@tinaok, yes the cell takes a considerable time to run. I've added a print statement for each loop.</p>

Reply:
closing conversation, resolved